### ISSUE TITLE
Release 1.1.1 on npm has console.log of all incoming strings

### ISSUE THREAD

**Issue by haizz** [borgar/textile-js#39]
textile.js, line 109:
```javascript
      // basic case
      if ( typeof jsonml === "string" ) {
        console.log( jsonml );
        return JSONML.escape( jsonml );
      }
```

Please remove as it piles up the log of an application with unnecessary info.

**Comment by haizz** [comment]
BTW git commit on tag 1.1.1 doesn't have this line.

**Comment by borgar** [comment]
Yep. That 1.1.1 package is totally broken. I don't even understand how I managed to mess the publishing up so bad. I will be pushing a 1.1.2 version to remedy this, 1.1.1 will remain broken because that is how NPM works. Hopefully the damage is minimal.

**Comment by borgar** [comment]
Ok. I have published a 1.1.2 version. The I have tested the NPM package and it behaves normally.
